frontend/src/app/organization/organization-details/organization-details.component.ts

```diff
@@ -13,4 +14,14 @@ export class OrganizationDetailsComponent {
   @Input() isUserOrgMember: boolean;
   @Input() organization: Organization;
   djangoRoot: string = environment.djangoRoot;
+
+  constructor(private metaService: Meta)
```

Comment:
this brackets format looks pretty bad. What do You think about
```
constructor(private metatagsService: MetatagsService) { }
```

Reply:
changed